x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/agg/Aggs.java

```diff
@@ -70,6 +70,11 @@ public Aggs(List<GroupByKey> groups, List<LeafAgg> simpleAggs, List<PipelineAgg>
         return groups;
     }
 
+    // For testing
+    public List<PipelineAgg> pipelineAggs() {
```

Comment:
`Aggs` does not expose its internal state - I would use instead the asAggBuilder() and its string to do the test matching.